apps/sim-core/README.md

```diff
@@ -12,4 +12,77 @@
+## Limitations
+
+The sim-core engine is strongly limited to being a run-only environment. Current recommended use is as follows:
+
+1) Run sim-core on localhost and view in your browser
+2) To open a simulation, use the 'import' functionality and target a .zip file containing a previously exported simluation.
+3) You can now run and edit this simulation, however file storage is simply maintained within your browser (using `localstorage`), and changes you make will only be preserved within this web browser.
+4) You can use the 'recent projects' menu to switch between other projects that you have imported.
+5) To experiment with an example project, import an example project .zip file from the `example_projects` folder.
+
+Please exercise caution if authoring work inside the self-hosted environment because any simulations you author are **not being presreved** outside of the browser environment.  These limitations will lift as the project status goals above are accomplished.
+
+## Installation
+
+Before running this software, your environment will need to have installed modern versions of:
+
+[Node](https://nodejs.org/en/), [Rust](https://www.rust-lang.org/learn/get-started), and [Yarn](https://yarnpkg.com/lang/en/).
+
+With these in place, you must use yarn to install wasm-pack:
+```sh
+yarn global add wasm-pack
+```
+
+To verify your installation, from the `sim-core` directory run
+```sh
+node -v
+yarn -v
+rustup default
+```
+If these commands output version numbers, you're all set.
+For the first build, simply run:
+```sh
+yarn
+```
+
+### Supported Environments
+
+The above requirements are all cross-platform, and as such sim-core can build and run in modern Windows, OSX, and Ubuntu, as well as within common VM's and containers.
+## Running `sim-core`
+
+To run the sim-core IDE, run:
+
+```sh
+yarn serve:core
+```
+
+This will compile the application and host it for you at a default location of [localhost:8080](http://localhost:8080)
+
+### Development and Troubleshooting
+
+See the README in [`packages/core`](https://github.com/hashintel/labs/tree/main/apps/sim-core/packages/core) for more details
+
+
+### Repository Structure
+
+Several different packages in this repository are orchistrated as yarn workspaces.  The key packages are:
+ - `core`, which is the React/Redux/Typescript frontend for the simulation engine. 
+ - `engine`, which is the rust implementation of the simulation engine
+ - `engine-web`, which bundles the rust-based `engine` into a webassembly-backed javascript interface using `wasm-bindgen`.
+
+ Additional utility packages also exist to facilitate minor conveniences.
+
+ While each package can be built and run separately using the `yarn` commands within its package (see the given package's package.json file for  guidance), in general they'll all be built together by running the `yarn` command from the `sim-core` folder.
```

Comment:
```suggestion
 While each package can be built and run separately using the `yarn` commands within its package (see the given package's `package.json` file for  guidance), in general they'll all be built together by running the `yarn` command from the `sim-core` folder.
```

I don't like saying "in general" here. Do we not always do this? Are there exceptions?

Reply:
I generally ( ;-) ) only ever run the full 'yarn' at installation, and run yarn serve:core as my dominant operation. I think that when developing, running 'yarn' is the exception rather than the common pattern.

I'll wordsmith a bit and push and update